### ISSUE TITLE
Catch errors does not work when using FireblocksWeb3Provider

### ISSUE THREAD

**Issue by emcpow2** [fireblocks/fireblocks-web3-provider#11]
Hi there, I'm trying to use `FireblocksWeb3Provider` and create error handling logic around transaction sending.

Apparently, if `FireblocksWeb3Provider` is used - then `try catch` section does not work as expected.

Code snipped below is a slightly modified version of example in [ethereum development](https://developers.fireblocks.com/docs/ethereum-development) official documentation.

```javascript
import {
  FireblocksWeb3Provider,
  ChainId,
} from "@fireblocks/fireblocks-web3-provider";
import * as ethers from "ethers";

// A Human-Readable ABI; for interacting with the contract, we
// must include any fragment we wish to use
const abi = [
  "function transferFrom(address from, address to, uint256 tokenId)",
];

// NFT Contract Address
const CONTRACT_ADDRESS = process.env.NFT_CONTRACT_ADDRESS;

const eip1193Provider = new FireblocksWeb3Provider({
  // apiBaseUrl: ApiBaseUrl.Sandbox, // If using a sandbox workspace
  privateKey: process.env.FIREBLOCKS_API_PRIVATE_KEY,
  apiKey: process.env.FIREBLOCKS_API_KEY,
  vaultAccountIds: process.env.FIREBLOCKS_VAULT_ACCOUNT_IDS,
  chainId: ChainId.MAINNET,
});

async function main() {
  const provider = new ethers.providers.Web3Provider(eip1193Provider);
  const signerAddress = process.env.FIREBLOCKS_VAULT_ADDRESS;
  const signer = provider.getSigner(signerAddress);

  const myContract = new ethers.Contract(CONTRACT_ADDRESS, abi, signer);
  const from = signerAddress;
  const to = "0x0000000000000000000000000000000000000000"; // random address

  // Invoke `transferFrom` method
  const tokenId = 2;
  let txnHash = "";
  try {
    txnHash = await myContract.transferFrom(from, to, tokenId);
  } catch (error) {
    console.log("error catch");
  }

  console.log("normal code execution");
}

main().catch((error) => {
  console.log(error);
});
```

I expect that `error catch` and the following `normal code execution` messages are printed in console. Instead I get the following:

```console
$ dotenv -- node --no-warnings src/main.js

/home/eduard/fireblocks-nft-transfer/node_modules/@fireblocks/fireblocks-web3-provider/dist/src/provider.js:166
                throw this.createError({ message: `No ${this.assetId} asset wallet found for vault account with id ${vaultAccountId}` });
                           ^

Error: No ETH asset wallet found for vault account with id 123456
    at FireblocksWeb3Provider.populateAccounts (/home/eduard/fireblocks-nft-transfer/node_modules/@fireblocks/fireblocks-web3-provider/dist/src/provider.js:166:28)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: -32603,
  data: undefined,
  payload: undefined
}

Node.js v18.12.0
```

There's a misconfiguration, but the `try catch` section should still work!

I suspect this there's a floating promise somewhere in library code. Probably in `FireblocksWeb3Provider` constructor.

## Software version

- `@fireblocks/fireblocks-web3-provider` `1.2.2`
- `ethers`: `5.7.2`
- Node.js 18.12.0
- Ubuntu 20.04 in WSL Linux

**Comment by emcpow2** [comment]
I tried a hack with explicitly wait for private `initialized` function. But it does not help.

```javascript
  try {
    await eip1193Provider["initialized"];
    txnHash = await myContract.transferFrom(from, to, tokenId);
  } catch (error) {
```

> Unfortunately the constructor can't be asynchronous in JS, and adding a mandatory 'init()' call is quite a big breaking change..

I need a way to handle the error, otherwise the API server simply crashes.

**Comment by orenyomtov** [comment]
Thanks for the PR, it has been merged and published as v1.2.3
